### DISCUSSION TITLE
Cannot filter on fields latitude and longitude from orm manager

### DISCUSSION THREAD

**Discussion by tux-mania** [netbox-community/netbox#14742] [Help Wanted!] [answered]
### Deployment Type

Self-hosted

### NetBox Version

v3.6.4

### Python Version

3.9

### Steps to Reproduce

1. Enter GPS Coordinates on one site (let's say site's ID is 2833) through the Web UI
2. Connect to the shell with 
```python
python manage.py nbshell
``` 
3. Select the site
```python
>>> site = Site.objects.get(id=2833)
```
4. Display latitude of this site
```python
>>> site.latitude
>>>
```

### Expected Behavior

The value from latitude field should be displayed.

### Observed Behavior

Nothing is shown, as if value of this field was None.
Same results happened using pynetbox (that's why I tested directly through the manager on Netbox).

**Comment by jeremystretch** [comment, marked as answer]
This works fine for me.

```python
>>> site = Site.objects.get(pk=2)
>>> site.latitude
Decimal('12.345000')
```

The only reason you would see anything returned accessing the `latitude` attribute is that no value is set on the object.

Are you sure you're connected to the same NetBox instance/database as the UI?

**Reply by tux-mania** [reply]
Hi Jeremy. 

Thanks for your answer.
Actually, you're right... I am working on 2 instances (as I cannot access the manager of the prod application).
My bad...
Still got an issue, but apparently pynetbox related.
Sorry to bother you for nothing!!!
